droidlet/perception/robot/tests/test_perception.py

```diff
@@ -36,15 +36,17 @@
 
 OFFICE_IMG_PATH = os.path.join(
     os.path.abspath(os.path.dirname(__file__)),
-    "test_assets",
+    "../../../../droidlet/artifacts/datasets/robot/perception_test_assets",
```

Comment:
Was thinking of how to make this prettier - found [this](https://stackoverflow.com/a/32698989) module that returns the repo root. We could define that variable and just access the artifacts without cd-ing up the directory tree.

Reply:
Perfect, yes this sounds good.